cmd/queue/main.go

```diff
@@ -139,14 +139,6 @@ func main() {
 	// Report stats on Go memory usage every 30 seconds.
 	metrics.MemStatsOrDie(ctx)
 
-	// Setup reporters and processes to handle stat reporting.
-	promStatReporter, err := queue.NewPrometheusStatsReporter(
```

Comment:
I think we can drop this reporter from the code base

Reply:
removed